src/test/java/robotbuilder/data/ExtensionFileValidationTest.java

```diff
@@ -0,0 +1,57 @@
+
+package robotbuilder.data;
+
+import org.junit.Before;
+import org.junit.Test;
+import robotbuilder.extensions.Extensions;
+
+import java.io.File;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author phurley
+ */
+public class ExtensionFileValidationTest {
+    Method isValidFile = null;
+
+    @Before
+    public void setUp() {
+        Method method = null;
+        try {
+            isValidFile = Extensions.class.getDeclaredMethod("isValidFile", File.class);
+            isValidFile.setAccessible(true);
+        } catch (NoSuchMethodException e) {
```

Comment:
Throw an assertion error, otherwise the tests will still "succeed" even though there was an exception.

`throw new AssertionError(e);` would be fine